models/CustomAlbum.js

```diff
@@ -1,22 +1,39 @@
 const mongoose = require("mongoose");
+const mongoosePaginate = require("mongoose-paginate-v2");
+
 const ratingSchema = require("./subDocuments/Rating");
+const { isValidUrl } = require("../routes/utils/validations");
+const { ERROR } = require("../constants/messages");
 
 const customAlbumSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+    required: true,
+    index: "hashed",
+  },
   category: {
     type: String,
     required: true,
-    unique: true,
   },
   url: {
     type: String,
+    validate: [ isValidUrl, ERROR.INVALID_URL],
```

Comment:
```suggestion
    validate: [isValidUrl, ERROR.INVALID_URL],
```

이전에 난 실수인데 저희가 확인을 못하게 넘어간 것 같습니다!
merge전에 수정 해주시면 감사하겠습니다.